src/coverage.ts

```diff
@@ -1,19 +1,19 @@
 export enum Cases {
-    caseOne = 1,
-    caseTwo = 2,
```

Comment:
I have prettier installed, so it automatically formatted this using the default config